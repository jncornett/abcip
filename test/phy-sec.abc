# phy is the default at layer 1 but we add it
# to make clear that there is a layer 1 proto
d ( stack="phy:eth:ip4:udp" )

c ( phy:sec=3.141592654, udp:a=1 )

a ( len=100 )
b ( len=100 )
a ( len=100 )

