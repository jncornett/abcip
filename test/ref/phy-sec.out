IP (tos 0x0, ttl 64, id 1, offset 0, flags [none], proto UDP (17), length 128)
    10.1.2.3.1 > 10.9.8.7.8: UDP, length 100
	0x0000:  0209 0807 0605 0201 0203 0405 0800 4500  ..............E.
	0x0010:  0080 0001 0000 4011 5c59 0a01 0203 0a09  ......@.\Y......
	0x0020:  0807 0001 0008 006c cd77 4c6f 7265 6d20  .......l.wLorem.
	0x0030:  6970 7375 6d20 646f 6c6f 7220 7369 7420  ipsum.dolor.sit.
	0x0040:  616d 6574 2c20 636f 6e73 6563 7465 7475  amet,.consectetu
	0x0050:  7220 6164 6970 6973 6369 6e67 2065 6c69  r.adipiscing.eli
	0x0060:  742e 2045 7469 616d 2073 7573 6369 7069  t..Etiam.suscipi
	0x0070:  742c 2071 7561 6d20 6174 2073 6365 6c65  t,.quam.at.scele
	0x0080:  7269 7371 7565 2075 6c74 7269 6369       risque.ultrici
IP (tos 0x0, ttl 64, id 1, offset 0, flags [none], proto UDP (17), length 128)
    10.9.8.7.8 > 10.1.2.3.1: UDP, length 100
	0x0000:  0201 0203 0405 0209 0807 0605 0800 4500  ..............E.
	0x0010:  0080 0001 0000 4011 5c59 0a09 0807 0a01  ......@.\Y......
	0x0020:  0203 0008 0001 006c cd77 4c6f 7265 6d20  .......l.wLorem.
	0x0030:  6970 7375 6d20 646f 6c6f 7220 7369 7420  ipsum.dolor.sit.
	0x0040:  616d 6574 2c20 636f 6e73 6563 7465 7475  amet,.consectetu
	0x0050:  7220 6164 6970 6973 6369 6e67 2065 6c69  r.adipiscing.eli
	0x0060:  742e 2045 7469 616d 2073 7573 6369 7069  t..Etiam.suscipi
	0x0070:  742c 2071 7561 6d20 6174 2073 6365 6c65  t,.quam.at.scele
	0x0080:  7269 7371 7565 2075 6c74 7269 6369       risque.ultrici
IP (tos 0x0, ttl 64, id 2, offset 0, flags [none], proto UDP (17), length 128)
    10.1.2.3.1 > 10.9.8.7.8: UDP, length 100
	0x0000:  0209 0807 0605 0201 0203 0405 0800 4500  ..............E.
	0x0010:  0080 0002 0000 4011 5c58 0a01 0203 0a09  ......@.\X......
	0x0020:  0807 0001 0008 006c e66d 6573 2c20 6e69  .......l.mes,.ni
	0x0030:  7369 206e 756c 6c61 2066 6572 6d65 6e74  si.nulla.ferment
	0x0040:  756d 206f 7263 692c 2061 2069 6d70 6572  um.orci,.a.imper
	0x0050:  6469 6574 206c 6163 7573 2065 6e69 6d20  diet.lacus.enim.
	0x0060:  6120 6665 6c69 732e 2043 756d 2073 6f63  a.felis..Cum.soc
	0x0070:  6969 7320 6e61 746f 7175 6520 7065 6e61  iis.natoque.pena
	0x0080:  7469 6275 7320 6574 206d 6167 6e69       tibus.et.magni
